run.py

```diff
@@ -1,32 +1,7 @@
-# -*- encoding: utf-8 -*-
-from __future__ import absolute_import, unicode_literals
-
-"""
-Copyright (c) 2020 - DevsBranch
-"""
-
-from sys import exit
 from app import create_app
 
-from decouple import config
-from config import config_dict
-
-# WARNING: Don't run with debug turned on in production!
-DEBUG = config("DEBUG", default=True)
-
-# The configuration
-get_config_mode = "Debug" if DEBUG else "Production"
-
-try:
-
-    # Load the configuration using the default values
-    app_config = config_dict[get_config_mode.capitalize()]
-
-except KeyError:
-    exit("Error: Invalid <config_mode>. Expected values [Debug, Production] ")
+app = create_app()
 
-app = create_app(app_config)
-# Migrate(app, db)
 
```

Comment:
could you explain why you trashed/moved content of this module ?

Reply:
I wanted to use the `create_app` both creating the for Flask application and the `celeryapp` package for Celery application without passing in the `config` argument into the `create_app` function. I have reverted the changes with just some slight changes. The `config` can now be created in each module, the `run.py` and `celery_worker.py` and pass onto the `create_app` function.

Reply:
okay
dont let the new libraries being introduce enforce you to change the structure of the application
that should be based only on architectural decision of the solution

lets keep it like that for now